packages/smart-contracts/scripts-create2/contract-setup/setups.ts

```diff
@@ -1,143 +1,10 @@
 import { HardhatRuntimeEnvironmentExtended } from '../types';
-import { erc20SwapConversionArtifact } from '../../src/lib';
-import { batchPaymentsArtifact } from '../../src/lib';
-import utils from '@requestnetwork/utils';
-import {
-  updateBatchPaymentFees,
-  updatePaymentErc20FeeProxy,
-  updatePaymentEthFeeProxy,
-  updateChainlinkConversionPath,
-  updateRequestSwapFees,
-  updateSwapRouter,
-} from './adminTasks';
+import { setupETHConversionProxy } from './setupETHConversionProxy';
+import { setupBatchPayments } from './setupBatchPayments';
+import { setupERC20SwapToConversion } from './setupERC20SwapToConversion';
 
 /**
- * Updates the values of the batch fees of the BatchPayments contract, if needed
- * @param contractAddress address of the BatchPayments Proxy
- * @param hre Hardhat runtime environment
- */
-const setupBatchPayments = async (
```

Comment:
FYI In such cases it's relevant to explain why you remove a method (defined twice, right?)

Reply:
I moved it to its own file